pontoon/checks/libraries/pontoon.py

```diff
@@ -1,11 +1,13 @@
 from __future__ import absolute_import
 
+import HTMLParser
 import re
 
+import bleach
+
 from collections import defaultdict
 from fluent.syntax import FluentParser, ast
 
-
```

Comment:
I would keep this blank line. As you can check in other files, we usually split imports and global variable definitions with two blank lines.